src/main/webapp/open_hours.js

```diff
@@ -45,36 +50,51 @@ class TimeOption {
     this.dayLabel.classList.add("day-label");
     this.dayOptionArea.appendChild(this.dayLabel);
 
-    this.dayOpenLabel = document.createElement("label");
-    this.dayOpenLabel.textContent = "Open";
-    this.dayOpenLabel.classList.add("day-open-label");
-    this.dayOptionArea.appendChild(this.dayOpenLabel);
-    this.dayOpenInput = document.createElement("input");
-    this.dayOpenInput.setAttribute("type", "radio");
-    this.dayOpenInput.setAttribute("name", `${this.day}-isOpen`);
-    this.dayOpenInput.setAttribute("value", "open");
-    this.dayOptionArea.appendChild(this.dayOpenInput);
-
-    this.dayClosedLabel = document.createElement("label");
-    this.dayClosedLabel.textContent = "Closed";
-    this.dayClosedLabel.classList.add("day-closed-label");
-    this.dayOptionArea.appendChild(this.dayClosedLabel);
-    this.dayClosedInput = document.createElement("input");
-    this.dayClosedInput.setAttribute("type", "radio");
-    this.dayClosedInput.setAttribute("name", `${this.day}-isOpen`);
-    this.dayClosedInput.setAttribute("value", "closed");
-    this.dayOptionArea.appendChild(this.dayClosedInput);
-
-    this.radioElements = [this.dayOpenInput, this.dayClosedInput];
-    if (this.forRegistration) {
-      this.dayOpenInput.setAttribute("checked", "checked");
-      this.timeInputArea.classList.add("show-time-area");
-    } else if (!this.organizationDay.propertyMap.isOpen) {
-      this.dayClosedInput.setAttribute("checked", "checked");
-      this.timeInputArea.classList.add("hide-time-area");
-    } else {
-      this.dayOpenInput.setAttribute("checked", "checked");
-      this.timeInputArea.classList.add("show-time-area");
+
+    // Only show open closed options for organization modification
```

Comment:
I'm not sure if having this for specific pages and not for other pages is the best way to solve reusing the TimeOption class, if we decide to go with this option.

For example, I was thinking about having the Event registration page having:

(MM/DD/YYYY)
Date                    Time(s)
07/20/2020         3:15 - 4:15
                           6:15 - 7:15
07/21/2020         5:00 - 7:00

Reply:
Right now it im designing it mvp style which would mean that the event would only have one day as an option so it is easier to implement. But they can have multiple times for that day